components/main-chef-wrapper/platform-lib/cli_helpers.go

```diff
@@ -0,0 +1,86 @@
+package platform_lib
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/chef/chef-workstation/components/main-chef-wrapper/dist"
+)
+
+var rubyenvMap map[string]interface{}
+
+func init() {
+	rubyenvMap = UnmarshallRubyEnv()
+}
+
+func PackageHome() string {
+	var packageHomeSet = os.Getenv("CHEF_WORKSTATION_HOME")
+	var packageHome string
+	if len(packageHomeSet) != 0 {
+		packageHome = packageHomeSet
+	} else {
+		packageHome = DefaultPackageName()
+	}
+	return packageHome
+}
+
+func DefaultPackageName() string {
+	// this logic can be used if other logic doesn't work.
+	//if runtime.GOOS == "windows" {
+	//home := os.Getenv("HOMEDRIVE") + os.Getenv("HOMEPATH")
+	//if home == "" {
+	//home = os.Getenv("USERPROFILE")
+	//home = os.Getenv("LOCALAPPDATA")
+	//}
+	//return home
+	//}
+	//return os.Getenv("HOME")
+	home, err := os.UserHomeDir()
+	if err != nil {
+		log.Fatal(err)
+	}
+	return filepath.Join(home, dist.WorkstationDir)
+}
+
+func OmnibusGemRoot() string {
+	gemRoot := rubyenvMap["omnibus path"].(map[string]interface{})["GEM_ROOT"]
```

Comment:
either define struct for ruby env or check before going next can we convert that interface into particular type 
data, ok := rubyenvMap["omnibus path"].(map[string]interface{})
if ok {
gemRoot := data["GEM_ROOT"]

}
like this in all func